src/commands/init.ts

```diff
@@ -134,20 +114,79 @@ export const validatePersonalAccessToken = async (
   }
 };
 
+export const isIntrospectionAzureDefined = (curConfig: ConfigYaml): boolean => {
+  if (!curConfig.introspection) {
+    return false;
+  }
+  const intro = curConfig.introspection;
+  return intro.azure !== undefined;
+};
+
+export const handleIntrospectionInteractive = async (
+  curConfig: ConfigYaml
+): Promise<void> => {
+  if (!isIntrospectionAzureDefined(curConfig)) {
+    curConfig.introspection = {
+      azure: {
+        key: new Promise(resolve => {
+          resolve(undefined);
+        })
+      }
+    };
+  }
+
+  // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
+  const azure = curConfig.introspection!.azure!;
+
+  const ans = await inquirer.prompt([
+    promptBuilder.azureStorageAccountName(azure.account_name),
+    promptBuilder.azureStorageTableName(azure.table_name),
+    promptBuilder.azureStoragePartitionKey(azure.partition_key),
+    promptBuilder.azureStorageRepoAccessKey(azure.source_repo_access_token),
+    promptBuilder.azureKeyVaultName(curConfig.key_vault_name)
+  ]);
+  azure["account_name"] = ans.azdo_storage_account_name;
+  azure["table_name"] = ans.azdo_storage_table_name;
+  azure["partition_key"] = ans.azdo_storage_partition_key;
+  azure["source_repo_access_token"] = ans.azdo_storage_repo_access_key;
```

Comment:
for now, we probably don't need to prompt for source_repo_access_token since this is more for github repos, we can use the access key from azure devops section for that. So same goes here, `source_repo_access_token` should not be associated with `azdo_storage_repo_access_key` since those are different fields

Reply:
accept
just make it clear the `source_repo_access_token` is just a key from prompt answer.
I will remove the prompt for `source_repo_access_token`